doc/api.md

```diff
@@ -124,6 +124,62 @@ Adds a row to the `mail_unsubscribes` table, preventing email form being sent to
 
 Requires an authentication token granting admin privileges. A request `Content-Type` of `application/x-www-form-urlencoded` is required. Renders the template with the data in the body.
 
+User Management
+---------------
+
+Most of these APIs operate on the data stored in members' records. The following mapping from database schema to JSON representation (as keys on a user object) is established for use with this API:
+ - `id`: The internal database ID is exposed as an integer `db_id`. This may not be null.
```

Comment:
I'm kinda :-1: on `null`s in json as opposed to the field being absent.

Reply:
The rationale for this is that we want to be able to represent
 - the DB contains some value x
 - the DB contains a NULL
 - this object (representing a search or a diff) doesn't have this set (doesn't want to update/search by this key)

Reply:
are IDs even allowed to be null?